llvm/test/CodeGen/AIE/aie2/sibling-constraint2.ll

```diff
@@ -17,13 +17,13 @@
 define void @test(i8 %a, i32 %b) {
 entry:
   %cmp0 = icmp eq i32 %b, 0
-  %sel0 = select i1 %cmp0, i8 1, i8 0
+  %sel0 = select i1 %cmp0, i8 0, i8 2
```

Comment:
Why this change?

Reply:
GlobalISel has a new combiner for G_SELECT, which eliminates selects based on 0/1. Using different values doesn't trigger these combines